agilent_infiniium_scope.py

```diff
@@ -17,7 +17,7 @@
 ### BEGIN NODE INFO
 [info]
 name = Agilent Infiniium Oscilloscope
-version = 0.2
+version = 0.2.1
```

Comment:
Given the scope of these changes (no pun intended), I would suggest making this `0.3.0`.